feat(serverless): call private functions and containers from triggers (#2293)
scaleway/docs-content: serverless/containers/how-to/add-trigger-to-a-container.mdx

</Message>

<Message type="important">

>   This feature is currently in beta.
> </Message>
> 
> <Message type="important">
>   It is possible to create triggers on private functions. But, if you want to **update** the privacy of a function from public to private, you will need to recreate the existing **SQS** and **NATS** triggers after the privacy change. We are actively working to make this operation as seamless as possible, but for now, you will need to recreate the triggers.

Reviewer: ```suggestion
  - This feature is currently in beta.

  - You can create triggers on private functions, but  to update the privacy of a function from **public** to **private**, you must recreate the existing **SQS** and **NATS** triggers after the privacy change. We are actively working to make this operation as seamless as possible.
```
